azurerm/internal/services/appconfiguration/app_configuration_resource.go

```diff
@@ -408,3 +447,30 @@ func flattenAppConfigurationAccessKey(input appconfiguration.APIKey) []interface
 		},
 	}
 }
+
+func expandAppConfigurationIdentity(d *schema.ResourceData) *appconfiguration.ResourceIdentity {
+	identities := d.Get("identity").([]interface{})
```

Comment:
Actually we never use the parameter `d` below after we have done this `d.Get`, how about we just use this `identities` of type `[]interface{}` as the parameter of this function?
```suggestion
func expandAppConfigurationIdentity(identities []interface{}) *appconfiguration.ResourceIdentity {
```